builtin/credential/ldap/path_config_rotate_root.go

```diff
@@ -0,0 +1,113 @@
+package ldap
```

Comment:
nit: might be worth adding in the copywrite headers to both new files so that we have CI completely green :) 

```
// Copyright (c) HashiCorp, Inc.
// SPDX-License-Identifier: MPL-2.0
```

Reply:
Will do - wasn't sure if we had a bot that would do it.

MPL is right here?

Reply:
Whoops my bad, copied that from the wrong repo. 😅 

Vault should be BUSL I believe. And yeah we do have a bot that would add it in a separate PR, definitely more of a nit for green CI 🟢 and not a blocker :)

```
// Copyright (c) HashiCorp, Inc.
// SPDX-License-Identifier: BUSL-1.1
```